pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/BrokerService.java

```diff
@@ -956,6 +960,14 @@ public void onUpdate(String path, Map<String, String> data, Stat stat) {
 
     private void updateDynamicServiceConfiguration() {
         try {
+            try {
+                // create znode if not present
+                byte[] data = ObjectMapperFactory.getThreadLocal().writeValueAsBytes(Maps.newHashMap());
```

Comment:
Instead of trying to create the z-node each time, we could first check if it exists and only trying to create it otherwise, still ignoring `NodeExistsException` to handle race conditions.